pages/blog.md

```diff
@@ -0,0 +1,33 @@
+--------------------------------
+
+After a fun ride of 7 years and 4 months, Nov 17th, 2022 was my last working day at Twitter by not choosing being “hardcore” ([context](https://gizmodo.com/elon-musk-email-be-part-of-hardcore-twitter-or-leave-1849789128)), so now is probably a good time to share part of what I have learned at Twitter (all “Twitter” references below are about the company, not the product).
+ 
+People have always said Twitter is an amazing place to work at, and culture is definitely part of that.
+
+### Communication
+
+What is culture? It can simply be how a group of people speak, ask questions, listen (if at all), give support, etc. Therefore, its culture affects how I behave, and at the same time, my behavior also contributes to such culture which affects other people.
+
+One of the core values is “communicate fearlessly to build trust”. To clarify, “fearless” doesn’t mean rude, but direct and timely. However if the receiver cannot process this correctly, things can go wrong very quickly. Personally I have observed disheartening cases like this where coworkers’ relationship soured for something totally repairable. Therefore it cannot be an empty statement, but a skill to learn.
+
+During my time at Twitter, I have assumed people always had the best intentions, and I have never felt disrespected even when my colleagues disagreed with me or felt embarrassed or ashamed admitting my own mistakes, including almost bringing down twitter.com a couple of times, because this is the basis to improve myself and our team. It is also a sign that other people have great trust in me when they are willing to provide the feedback.
+
+Twitter also puts these values into action. For example, TWIG was a training program for seasoned engineers and managers to learn about what kind of people exist on this planet and what the best way is to communicate with them. “Recognize that passion and personality matter.”
+
+One key takeaway for me is that trust is the foundation for any communication. It does not matter whether what you said was right or wrong, if the receiver has no trust in the giver, there is likely no positive outcome. This is why I think it is extremely important to understand why things are the way they are or people behave the way they do before making any suggestions or judgments. Therefore it is essential to explicitly state the intention and to set up a common goal before any difficult conversation.
+
+### Growing with Others
+
+It is easy to see the benefit of working with more experienced people, but I want to highlight that the reserve is also true. 
+
+I have always enjoyed mentoring and working with junior engineers (in tenure or experience). It may be a natural instinct that engineers don’t like others to touch their code, but sometimes our instincts serve us the opposite way. For example, it is natural to panic after getting bit by a venomous snake, but it will in turn make their blood flow faster and sustain more damage. 
```

Comment:
```suggestion
I have always enjoyed mentoring and working with junior engineers (in terms of tenure or experience). It may be a natural instinct that engineers don’t like others to touch their code, but sometimes our instincts serve us the opposite way. For example, it is natural to panic after getting bit by a venomous snake, but it will in turn make their blood flow faster and sustain more damage. 
```